CardData.py

```diff
@@ -51,82 +51,11 @@
 INVALID_TOKENS = ['200175', '200176']
 
 """
- Categories are stored as sums of powers of 2. Use the ID from here to look up
- localisations in categories output file.
+ WARNING : don't change it for a simple range(1,MAX), some category numbers aren't here.
```

Comment:
Leave this python doc comment as

`Use the ID from here to look up localisations in categories output file.`

Move this warning to a comment above the line with `range(LAST_CATEGORY, 1, -1)`